serverless/aws/examples/src/main/scalajvm/sttp/tapir/serverless/aws/examples/SamJsTemplateExample.scala

```diff
@@ -0,0 +1,30 @@
+package sttp.tapir.serverless.aws.examples
+
+import sttp.tapir.serverless.aws.examples.LambdaApiExample.helloEndpoint
+import sttp.tapir.serverless.aws.sam.{AwsSamInterpreter, AwsSamOptions, CodeSource}
+
+import java.nio.charset.StandardCharsets.UTF_8
+import java.nio.file.{Files, Paths}
+
+/** Before running the actual example we need to interpret our api as SAM template */
+object SamJsTemplateExample extends App {
```

Comment:
hm does it make sense to run the template generation using js? it produces the same output as the jvm one, and it's a build-time thing anyway?

Reply:
JS linker won't build it due to missing YAML support. Besides, users run the examples via sbt anyway.

Reply:
ah it's in the examples. Yeah, that's what I meant, that there's no point moving this to js, all good :)